runtime/jcl/common/java_lang_invoke_MethodHandleNatives.cpp

```diff
@@ -1366,13 +1366,8 @@ Java_java_lang_invoke_MethodHandleNatives_resolve(
 					target = (jlong)offset;
 				}
 			}
-			if (
-#if defined(J9VM_OPT_VALHALLA_VALUE_TYPES)
-				/* In project Valhalla fields may start at offset 0. */
-				(NULL != new_clazz)
-#else /* defined(J9VM_OPT_VALHALLA_VALUE_TYPES) */
-				(0 != target)
-#endif /* defined(J9VM_OPT_VALHALLA_VALUE_TYPES) */
+			/* A field may start at offset 0. */
```

Comment:
There is no check for the offset now. So probably better to update this comment to sth like:
"A field may start at offset 0, so no need to check if 0 != target"

Reply:
Updated